src/components/TodoModal/TodoModal.tsx

```diff
@@ -15,30 +39,36 @@ export const TodoModal: React.FC = () => {
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
             >
-              Todo #2
+              {`Todo #${todoSelected.id}`}
             </div>
 
             {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
             <button
               type="button"
               className="delete"
               data-cy="modal-close"
+              onClick={() => onSelect(undefined)}
             />
           </header>
 
           <div className="modal-card-body">
             <p className="block" data-cy="modal-title">
-              quis ut nam facilis et officia qui
+              {todoSelected.title}
             </p>
 
             <p className="block" data-cy="modal-user">
-              {/* <strong className="has-text-success">Done</strong> */}
-              <strong className="has-text-danger">Planned</strong>
+              {todoSelected.completed
+                ? (
+                  <strong className="has-text-success">Done</strong>
+                )
+                : (
+                  <strong className="has-text-danger">Planned</strong>
+                )}
```

Comment:
we can use single `strong` tag and only change `success` to `danger` and children